RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@ Unreleased
 * [**] Fixes a crash related to HTML to blocks conversion when no transformations are available [https://github.com/WordPress/gutenberg/pull/56723]
 * [**] Fixes a crash related to undefined attributes in `getFormatColors` function of `RichText` component [https://github.com/WordPress/gutenberg/pull/56684]
 * [**] Fixes an issue with custom color variables not being parsed when using global styles [https://github.com/WordPress/gutenberg/pull/56752]
+* [**] Fixes a crash when blockType wrapperProps are not defined [https://github.com/WordPress/gutenberg/pull/56846]
```

Comment:
This entry should be under the `Unreleased` section as it will be released in the next version, not version `1.109.2`. I'll update the release notes accordingly.